packages/jobs/README.md

```diff
@@ -3,3 +3,6 @@
 Formerly known as alice-eth. It is currently coupled very
 tightly with Alice production infrastructure, so tests cannot
 be run without secrets for Alice sandbox environments.
+
+# Environmental config variables
+Please copy .envrc file and configure direnv tool https://direnv.net/
```

Comment:
Could you please paste what commands have you used to install it on mac so we save time on reading through docs and experimenting with different packaging tools?